custom/icds_reports/management/commands/sql_scripts/extract_valid_phone_count_grouped_by_district.sql

```diff
@@ -0,0 +1,108 @@
+SELECT 
+person_cases.district_id,
+person_cases.state_id,
+-- Pregnant Women
+SUM(
+    CASE 
+        WHEN is_pregnant = 1 
+            THEN 1 
+        ELSE 0
+    END) "pw_count",
+SUM(
+    CASE 
+        WHEN is_pregnant = 1 
+        AND phone_number similar to '91[6789][0-9]{9}' THEN 1 else 0
```

Comment:
you sure phone_number property is preappended with `91`?

Reply:
Yes. I am. Actually phone number validation has been done before and https://dimagi-dev.atlassian.net/browse/ICDS-1325 ticket mentions the criteria for the validation.